cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -612,37 +566,60 @@ private void assertConsistentTypes(Class<? extends Mapper> firstMapperClass,
 
   /**
    * Sets the configurations used for inputs.
+   * Multiple mappers could be defined, so we first check that their output types are consistent.
+   *
+   * @return the TypeToken for one of the mappers (doesn't matter which one, since we check that all of their output
+   * key/value types are consistent.
```

Comment:
Er, I fixed the javadoc to not mention that it returns null anymore; just forgot to remove the annotation `@Nullable`. Removed it now.